support/macos/PrivacyInfo.xcprivacy

```diff
@@ -6,7 +6,14 @@
 	<array/>
 	<key>NSPrivacyAccessedAPITypes</key>
 	<array>
-		<dict/>
+		<dict>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>DDA9.1</string>
```

Comment:
I'm not sure if DDA9.1 is the right code here

![Screenshot 2024-04-13 at 09 35 48](https://github.com/krzyzanowskim/OpenSSL/assets/758033/fb1cd963-8603-425e-859a-f0730588fb3c)

Reply:
you're right, DDA9.1 is specific for timestamps, C617.1 seems to be the correct reason. 

e.i.
https://github.com/openssl/openssl/blob/d4188f24866f88b4269110ce86f9545edd44c846/apps/lib/apps.c#L1688
https://github.com/openssl/openssl/blob/d4188f24866f88b4269110ce86f9545edd44c846/apps/lib/opt.c#L1272C5-L1275C19

Reply:
these are apps, cli apps, that are not part of the libraries. I have no evidence library uses any of that so far, that makes this privacy change request invalid.

Reply:
Hi @krzyzanowskim, OpenSSL is on the list of SDKs that require the privacy manifest, provided by Apple:
https://developer.apple.com/support/third-party-SDK-requirements/

Reply:
it already comes with privacy manifest and it is completely valid. I don’t think the changes to the manifest proposed in this PR are valid.